### ISSUE TITLE
Using "per month" as a standalone localization string

### ISSUE THREAD

**Issue by tofumatt** [mozilla/payments-ui#296]
In https://github.com/mozilla/payments-ui/blob/master/public/js/components/subscription.jsx#L27 we use `{productPrice} {gettext('per month')}`, but it seems like a localized string like `{gettext('{price} per month', {price: productPrice})}` (I don't know the actual API) would be better, as some languages may not write prices per month in that format.

I don't have a language in mind that violates the convention, but I feel like this is a bug I've run into before.

If this is valid, feel free to assing the bug to me :wink:

**Comment by muffinresearch** [comment]
Yep, we dont' currently have formatting. It would need to be something like: `format(gettext('{price} per month'), {price: productPrice})` to not break the extraction. E.g. formatting would need to be outside of the gettext func.
